extensions/amp-image-slider/0.1/test/integration/test-amp-image-slider.js

```diff
@@ -60,50 +66,760 @@ config.run('amp-image-slider', function() {
+          // Slider bar stays at original place (leftmost this case)
+          return verifyPositionAfterTimeout(s1, s1.pos.percent0);
+        }).then(() => {
+          // We focus again here!
+          s1.slider.focus();
+          return verifyPositionUpdateAfterEventDispatch(
+              /*eventConfig*/
+              {
+                target: s1.slider,
+                cstr: createKeyDownEvent,
+                key: 'Home',
+              },
+              /*observeTarget*/s1.slider,
+              /*sliderInfo*/s1,
+              /*targetPos*/s1.pos.percent50
+          );
+        });
+      });
+
+      it('pressing PageDown should move slider bar to rightmost ' +
+        'only when slider is focused', () => {
+        // Notice that we are not focusing on the slider here
+        s1.slider.dispatchEvent(createKeyDownEvent('PageDown'));
+        // Slider bar stays at original place (center by default)
+        return verifyPositionAfterTimeout(s1, s1.pos.percent50).then(() => {
+          // We focus here!
+          s1.slider.focus();
+          return verifyPositionUpdateAfterEventDispatch(
+              /*eventConfig*/
+              {
+                target: s1.slider,
+                cstr: createKeyDownEvent,
+                key: 'PageDown',
+              },
+              /*observeTarget*/s1.slider,
+              /*sliderInfo*/s1,
+              /*targetPos*/s1.pos.percent100
+          );
+        });
+      });
+    });
+
+    it('should seekTo correct position', () => {
+      // Button for seeking to 40% from left
+      const b1 = doc.querySelector('#b1');
+      const clickButtonFunction =
+        () => b1.click();
+      const observerCallback =
+        () => hasCorrectSliderPosition(s1, s1.pos.percent40);
+      // Click the button and observe slider bar position update
+      return invokeAndObserve(
+          /*invokeFunc*/clickButtonFunction,
+          /*target*/s1.slider,
+          /*cb*/observerCallback,
+          /*opt_errorMessage*/`Failed to seek to ${s1.pos.percent40}`
+      );
+    });
+
+    describe('hint behavior', () => {
+      it('should hide hint on user interaction (e.g. mousedown)', () => {
+        const dispatchMouseDownEventFunction =
+          () => s1.slider.dispatchEvent(createMouseDownEvent(s1.pos.percent40));
+        const isHintHiddenCallback =
+          () => s1.leftHint.classList
+              .contains('i-amphtml-image-slider-hint-hidden');
+        // Initially hint should be displayed
+        expect(isHintHiddenCallback()).to.be.false;
+        // Make sure hint is hidden after interaction
+        return invokeAndObserve(
+            /*invokeFunc*/dispatchMouseDownEventFunction,
+            /*target*/s1.slider,
+            /*cb*/isHintHiddenCallback,
+            /*opt_errorMessage*/'Hint failed to be hidden'
+        );
+      });
+
+      it('should show hint again on slider scrolling back and into viewport, ' +
```

Comment:
Since we know that this is flakey, can we skip it in this PR and re-enable it when we fix it?

Reply:
Seems no easy way to fix it without introducing extra code to extension source. Skipping for now.